cpp/include/cudf/lists/lists_column_device_view.cuh

```diff
@@ -22,8 +22,6 @@
 
 namespace cudf {
 
-namespace detail {
```

Comment:
Why was this removed? Is this now intended to be a publicly consumable type?

Reply:
I observed that most other `*_device_view` classes are not in the `detail::` namespace thus want to make things more consistent.

Reply:
There are many things that are not in a detail namespace that should be. This is moving in the wrong direction.

Reply:
Okay then I'll reverse it. Next time will move things into `detail::` instead.